chef/cookbooks/neutron/recipes/common_agent.rb

```diff
@@ -49,19 +49,20 @@
   subscribes :run, resources(:cookbook_file=> disable_rp_filter_file), :delayed
 end
 
+if neutron[:neutron][:networking_plugin] == "ml2"
+  ml2_mech_drivers = neutron[:neutron][:ml2_mechanism_drivers]
+  case
+  when ml2_mech_drivers.include?("openvswitch")
+    neutron_agent = node[:neutron][:platform][:ovs_agent_name]
+    neutron_agent_pkg = node[:neutron][:platform][:ovs_agent_pkg]
+    agent_config_path = "/etc/neutron/plugins/openvswitch/ovs_neutron_plugin.ini"
+  when ml2_mech_drivers.include?("linuxbridge")
+    neutron_agent = node[:neutron][:platform][:lb_agent_name]
+    neutron_agent_pkg = node[:neutron][:platform][:lb_agent_pkg]
+    agent_config_path = "/etc/neutron/plugins/linuxbridge/linuxbridge_conf.ini"
+  end
```

Comment:
So if we specify `mech_drivers = ["openvswitch", "linuxbridge"]`, when will the linuxbridge agent ever be installed?